packages-exp/installations-exp/src/api/get-installations.ts

```diff
@@ -19,7 +19,8 @@ import { FirebaseApp, getApp, _getProvider } from '@firebase/app-exp';
 import { Installations } from '../interfaces/public-types';
 
 /**
- * Returns an instance of FirebaseInstallations associated with the given FirebaseApp instance.
+ * Returns an instance of Firebase Installations associated with the given
```

Comment:
In long discussions when Andreas was still around, we decided not to capitalize "installations," even in the product name, unless heading case or something called for it -- so all of these instances should be lowercase.

These are the names/formats we agreed on: https://source.corp.google.com/piper///depot/google3/third_party/devsite/firebase/en/_internal/includes/_names.html;ws=codereview%2F392183348;rcl=378662242;cl=2;l=241

Reply:
It's referring to an instance of the SDK service so I went with `{@link Installations}`.